packer/install-dependencies.sh

```diff
@@ -350,7 +350,7 @@ K8S_VERSIONS="
 1.14.0
 1.13.5
 1.13.4
-1.12.7
+1.12.7-azs
```

Comment:
this should be additive, we want both `1.12.7` and `1.12.7-azs`

Reply:
yeah, sorry